### PR TITLE
Update triggers logic

### PR DESCRIPTION
**Pull request by kostyanf14**

### CODE REVIEW

File: triggers.yml.example [hunk 1]
```diff
@@ -1,40 +1,120 @@
...
+    - vioserial/lib/
+    - vioserial/libtestapp/
+    - vioserial/libtestsvc/
+    - vioserial/LICENSE
 viosock:
-  - 'viosock'
+  include:
+    - viosock/
+  exclude:
+    - viosock/ViosockPackage/
```
Comment (YanVugenfirer) [inline triggers.yml.example:94]: Not sure about excluding the package.

File: triggers.yml.example [hunk 2]
```diff
@@ -1,40 +1,120 @@
...
+  include:
+    - viostor/
+  exclude:
+    - viostor/LICENSE
+"*":
+  include:
+    - /
+    - Tools/
+    - VirtIO/
+    - packaging/
```
Comment (YanVugenfirer) [inline triggers.yml.example:110]: Same here. I think packaging projects should be included.

**Review by YanVugenfirer — APPROVED** [review]

File: triggers.yml.example (hunk 2 above)
Comment (YanVugenfirer) [reply triggers.yml.example:110]: We discussed it off the list. Packaging projects should be added.

**Review by YanVugenfirer — CHANGES_REQUESTED** [review]
Do not exclude packaging projects

File: triggers.yml.example (hunk 2 above)
Comment (kostyanf14) [reply triggers.yml.example]: Fixed

File: triggers.yml.example (hunk 1 above)
Comment (kostyanf14) [reply triggers.yml.example]: Fixed

**Review by YanVugenfirer — APPROVED** [review]

**Pull request merged**